modules/candidate_parameters/jsx/ExternalIdentifier.js

```diff
@@ -0,0 +1,350 @@
+/* exported RExternalIdentifier */
+
+var ExternalIdentifier = React.createClass({
+  getInitialState: function() {
+    return {
+      Data: [],
+      formData: {},
+      existingStudies: [],
+      updateResult: null,
+      errorMessage: null,
+      isLoaded: false,
+      loadedData: 0
+    };
+  },
+  componentDidMount: function() {
+    this.fetchData();
+  },
+  fetchData: function() {
+    var that = this;
+    $.ajax(
+      this.props.dataURL,
+      {
+        dataType: 'json',
+        xhr: function() {
+          var xhr = new window.XMLHttpRequest();
+          xhr.addEventListener(
+            "progress",
+            function(evt) {
+              that.setState({
+                loadedData: evt.loaded
+              });
+            });
+          return xhr;
+        },
+        success: function(data) {
+          that.setState({
+            Data: data,
+            isLoaded: true,
+            existingStudies: data.existingStudies
+          });
+        },
+        error: function(data, errorCode, errorMsg) {
+          that.setState({
+            error: 'An error occurred when loading the form!'
+          });
+        }
+      }
+    );
+  },
+  setFormData: function(formElement, value) {
+    var formData = this.state.formData;
+    formData[formElement] = value;
+    this.setState({
+      formData: formData
+    });
+  },
+  onSubmit: function(e) {
+    e.preventDefault();
+  },
+  render: function() {
+    if (!this.state.isLoaded) {
+      if (this.state.error !== undefined) {
+        return (
+          <div className="alert alert-danger text-center">
+            <strong>
+              {this.state.error}
+            </strong>
+          </div>
+        );
+      }
+
+      return (
+        <button className="btn-info has-spinner">
+          Loading
+          <span
+            className="glyphicon glyphicon-refresh
```

Comment:
GitHub highlights this as an error (even though it's not..) can you move the 2 glass names onto the same line so it doesn't get highlighted?